test/tests/multiapps/steffensen/steady_main.i

```diff
@@ -63,6 +63,7 @@
   fixed_point_algorithm = 'steffensen'
   fixed_point_max_its = 30
   transformed_variables = 'u'
+  accept_on_max_fixed_point_iteration = true
```

Comment:
this test is also badly designed. It's weird that you are not getting the same issue with the secant method. I ll update this one.

Reply:
Sounds great!